game/static/game/css/board.css

```
@@ -459,292 +215,91 @@ body {
+    font-size: 0.65rem;
+    letter-spacing: 1px;
+    font-weight: 700;
+    opacity: 0.7;
 }
-.btn-pause:hover {
-    background: linear-gradient(135deg, #cc9a10, #a07808);
-    transform: translateY(-1px);
-    box-shadow: 0 4px 16px rgba(184, 134, 11, 0.3);
+
+.clock .time {
+    font-family: 'Consolas', 'Courier New', monospace;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)